app/templates/buyers/preview_brief_source.html

```diff
@@ -52,99 +52,78 @@
-          {"label": "Published", "value": important_dates.published_date|dateformat},
-          {"label": "Deadline for asking questions", "value": important_dates.questions_close|utcdatetimeformat},
-          {"label": "Closing date for applications", "value": important_dates.closing_date|utcdatetimeformat}
-        ],
-        caption="Important dates",
-        field_headings=[
-          "Opportunity attribute name",
-          "Opportunity attribute value"
-        ],
-        field_headings_visible=False
-      ) %}
-      {% call summary.row() %}
-        {{ summary.field_name(item.label) }}
-        {{ summary.text(item.value) }}
-      {% endcall %}
-    {% endcall %}
-
-    {% for section in content.summary(brief) %}
-      {{ summary.heading(section.name, id="opportunity-attributes-{}".format(loop.index)) }}
-      {% if section.summary_page_description %}
-        {{ summary.description(section.summary_page_description) }}
-      {% endif %}
-      {% call(item) summary.list_table(
-        section.questions,
-        caption=section.name,
-        field_headings=[
-          "Opportunity attribute name",
-          "Opportunity attribute value"
+      <h2 class="govuk-visually-hidden">Important dates</h2>
+      {{ govukSummaryList({
+        "rows": [
+          {
+            "key": {"text": "Published"},
+            "value": {"text": important_dates.published_date|dateformat}
+          },
+          {
+            "key": {"text": "Deadline for asking questions"},
+            "value": {"text": important_dates.questions_close|utcdatetimeformat}
+          },
+          {
+            "key": {"text": "Closing date for applications"},
+            "value": {"text": important_dates.closing_date|utcdatetimeformat}
+          }
         ],
-        field_headings_visible=False
-      ) %}
-        {% call summary.row() %}
-          {{ summary.field_name(item.label) }}
-          {% with item_value = summary[item.type](item.value) | format_links(open_links_in_new_tab=True) %}
-              {{ (item_value | preserve_line_breaks) if item.type == 'textbox_large' else item_value }}
-          {% endwith %}
-        {% endcall %}
-      {% endcall %}
-  {% endfor %}
-
+        "attributes": {
+          "id": "opportunity-important-dates"
+        }, 
+        "classes": 'app-govuk-summary-list app-govuk-summary-list--top-border govuk-!-margin-bottom-8'
+      }) }}
+
+      {% for section in content_summary %}
+        <h2 id="{{ 'opportunity-attributes-{}'.format(loop.index) }}" class="govuk-heading-m app-summary-list-heading">{{ section.name }}</h2>
+        {% if section.summary_page_description %}
+          <p class="govuk-body govuk-!-margin-top-3">{{ section.summary_page_description }}</p>
+          {# DISPLAY MANDATORY EVALUATION METHODS #}
+        {% elif section.name == 'How suppliers will be evaluated' and evaluation_description %}
+          <p class="govuk-body govuk-!-margin-top-3">{{ evaluation_description }}</p>
```

Comment:
I don't think we're actually supplying `evaluation_description` in the context for this view 😿 That's a separate bug, I think (for DOS4 briefs only, the "How suppliers will be evaluated" section should have a description of either 'All suppliers will be asked to provide a written proposal' or 'All suppliers will be asked to provide a work history', depending on the lot). 

It's up to you whether you want to keep this logic in the template here, or take it out and wait until the fix is in (I'll create a separate card for that).

Reply:
I've jettisoned it and added a comment to the ticket!